### PR TITLE
strings

### PR DESCRIPTION
**Pull request by ZoyaNaumenko**

### CODE REVIEW

File: Exercises/strings/strings.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+console.log(smile);
+
+//2. Declare a variable with string with escaped symbols.
+
+const hello = 'Hello. I\'m Zoya';
+console.log(hello);
+
+// 3. Declare a variable which will contain name of your favourite animal.
+
+const myFavoriteAnimal = 'cat';
```
Comment (lizarusi) [inline Exercises/strings/strings.js:13]: mine too!

File: Exercises/strings/strings.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+// 5. Declare a variable which contain name and surname of your favourite writer in lowercase. Log his/her name with capitals first letters.
+
+const firstName = 'neil';
+const lastName = 'gaiman';
+
+const firstLetter1 = firstName[0].toUpperCase();
+const firstLetter2 = lastName[0].toUpperCase();
+
+console.log(firstLetter1 + firstName.slice(1) +' '+ firstLetter2 + lastName.slice(1));
```
Comment (lizarusi) [inline Exercises/strings/strings.js]: try to use back-ticks here

File: Exercises/strings/strings.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+// # Additional
+
+// 1. Declare a function which will receive password string. Check is password strong enough:
+//   - should contain at least one capital letter;
+//   - length no less than 8 characters.
+//   - should contain at leas one special character.
+
+function passvordValidation(pwd){
+
```
Comment (lizarusi) [inline Exercises/strings/strings.js:38]: redundant space

File: Exercises/strings/strings.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    console.log('Pasword should include at least 1 LowCase letter');
+    return false;
+  }
+      
+  if (regexp.test(pwd)) {
+    console.log('Pasword should be include at least 1 special character');
+    return false;
+  }
+      
+  if (pwd.length < 8) {
```
Comment (lizarusi) [inline Exercises/strings/strings.js:50]: better to put faster checks to top, because regex perform much longer than check on length

**Review by lizarusi — CHANGES_REQUESTED** [review]
very nice!

File: Exercises/strings/strings.js (hunk 1 above)
Comment (ZoyaNaumenko) [reply Exercises/strings/strings.js:13]: 😻

**Review by Mike-Savin — APPROVED** [review]

**Pull request merged**